### ISSUE TITLE
Link Text is Unclear

### ISSUE THREAD

**Issue by josephrorr** [Khan/tota11y#160]
Hello! Great plugin and logo ;)

I'm doing accessibility cleanup at https://www.quogueclub.com, running the tota11y chrome plugin and noticing one error (repeated for my main navigation) that I cannot get rid of! I've given each nav item a label, title, title attribute and description, and still getting an error:

Link text "null" is unclear / Link Text is Unclear

I'm curious to see if anyone in the community knows how I can fix this issue. It's the last issue in my ADA compliance update for the homepage. Any recommendations would be great appreciated! The issue can be reproduced by visiting https://www.quogueclub.com and running the tota11y checker on this page...other pages will be updated soon.

**Comment by josephrorr** [comment]
I think I found the issue. It's with font awesome's <i> tags not having any text in between.
